bundle/config/resources/job.go

```diff
@@ -44,3 +45,18 @@ func (j *Job) Exists(ctx context.Context, w *databricks.WorkspaceClient, id stri
 func (j *Job) TerraformResourceName() string {
 	return "databricks_job"
 }
+
+func (j *Job) InitializeURL(urlPrefix string, urlSuffix string) {
```

Comment:
Note that this PR also shows the `URL` in the JSON output of `databricks bunde summary --output json`. Because of that I needed to have a separate URL field / GetURL() getter / InitializeURL() method.